### PR TITLE
Dom with fetch ng

### PR DESCRIPTION
**Pull request by taylorjohnson141**
## Description

Clearly and concisely describe the feature.
This feature adds the ability to randomize a user, the ability to see the users pantry in Pantry.html and refactors existing methods on the Scripts.js and the DomUpdates.js

## Affected areas of application
Pantry.html, Srcipts.js DomUpdates.js 

List all parts of the website that will be altered from this PR.

Pantry.html, Srcipts.js DomUpdates.js 

## How Has This Been Tested?
This has been tested by checking the UI. 

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- Test A

- Test B

## Relevant Tickets

What tickets/issues were resolved by this feature

- Closes <...>

## Screenshots

Add screenshots here if applicable

Screen Shot 2020-09-12 at 7 36 07 PM

## Other considerations 
We may need to make two different scripts files to separate the index.html and the Pantry.html when things are undefined in one they mess up the code.
We have not used any classes yet and will need to in the future.

### CODE REVIEW

File: src/domUpdates.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+let domUpdates = {
+  randomNumber :  Math.floor(Math.random() * 49) + 1,
```
Comment (nicolegooden) [inline src/domUpdates.js:2]: I like this here!

File: src/scripts.js [hunk 2]
```diff
@@ -1,168 +1,213 @@
-import './css/base.scss';
-import './css/styles.scss';
+// import './css/base.scss';
+// import './css/styles.scss';
 
-import recipeData from './data/recipes';
-import ingredientData from './data/ingredients';
-import users from './data/users';
+// import recipeData from './data/recipes';
```
Comment (nicolegooden) [inline src/scripts.js:4]: I think having the `scripts` in `index.html` is fine but we'll have to use `import/export` syntax later.

File: src/scripts.js [hunk 3]
```diff
@@ -1,168 +1,213 @@
...
+window.onload = function() {
+  domUpdates.grabRecipes()
+  // domUpdates.grabUsers()
+  domUpdates.mergeFetchTimelines()
 }
+// homeButton.addEventListener('click', cardButtonConditionals);
+// favButton.addEventListener('click', viewFavorites);
+// cardArea.addEventListener('click', cardButtonConditionals);
 
-function viewFavorites() {
```
Comment (nicolegooden) [inline src/scripts.js:36]: this function is A LOT! Glad you destroyed it.

File: src/scripts.js [hunk 4]
```diff
@@ -1,168 +1,213 @@
...
 
-function getFavorites() {
-  if (user.favoriteRecipes.length) {
-    user.favoriteRecipes.forEach(recipe => {
-      document.querySelector(`.favorite${recipe.id}`).classList.add('favorite-active')
-    })
-  } else return
-}
 
-function populateCards(recipes) {
```
Comment (nicolegooden) [inline src/scripts.js:143]: Thanks for removing this.  Pretty sure we are doing this already with one of our other functions in a cleaner way.  Cards are being populated with `displayAllRecipes()`

**Review by nicolegooden — APPROVED** [review]
Hey @taylorjohnson141 ! This looks good overall and we should review the purpose of those other functions in `scripts.js` today.  We need to filter through what should be kept, modified, and removed completely.  Some of the functions have useful code, but the query selectors need to be checked.

File: src/index.html [hunk 5]
```diff
@@ -4,12 +4,12 @@
     <meta charset='UTF-8'>
     <title>Whats Cookin</title>
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap' rel='stylesheet'>
-    <link rel="stylesheet" href="./pantry.html">
+    <link rel="page" href="./pantry.html">
```
Comment (CalebCyphers) [inline src/index.html:7]: Good catch here

File: src/index.html [hunk 6]
```diff
@@ -22,7 +22,7 @@ <h1 class="open-menu-title x">Menu</h1>
     </header>
     <section class="open-menu-selections x">
       <article class="menu-my-pantry menu-selection x">
-        <h3 class="menu-my-pantry-title">My Pantry</h2>
+        <h2 class="menu-my-pantry-title"><a href="./pantry.html">My Pantry</a></h2>
```
Comment (CalebCyphers) [inline src/index.html:25]: Oh, this is a good change, I'll adjust the styles to reflect the links.

File: src/index.html [hunk 7]
```diff
@@ -42,20 +42,24 @@ <h1 class="page-title">All Recipes</h1>
...
           </article>
-        </article>
+        </article> -->
       </section>
-    </main>
+    </main> <script type='text/javascript' src='./pantry.js'></script>
+    <script type='text/javascript' src='./user.js'></script>
+    <script type='text/javascript' src='./recipe.js'></script>
+    <script type='text/javascript' src='./domUpdates.js'></script>
+    <script type='text/javascript' src='./scripts.js'></script>
```
Comment (CalebCyphers) [inline src/index.html:63]: We'll need to refactor these to 'imports' eventually.

**Review by CalebCyphers — APPROVED** [review]

**Pull request merged**